phantazm-mob/src/main/java/com/github/phantazmnetwork/mob/MobModel.java

```diff
@@ -46,14 +48,15 @@ public class MobModel implements Keyed {
     public MobModel(@NotNull Key key, @NotNull MinestomDescriptor descriptor,
                     @NotNull Collection<Collection<Goal>> goalGroups,
                     @NotNull Map<Key, Collection<Skill>> triggers, @Nullable Component displayName,
-                    @NotNull Map<EquipmentSlot, ItemStack> equipment, float maxHealth) {
+                    @NotNull Map<EquipmentSlot, ItemStack> equipment, float maxHealth, float speed) {
```

Comment:
speed and maxHealth are both attributes, have you considered a general way to serialize all mob attributes so they can be modified via config?